fluent/reference.md

```diff
@@ -618,14 +618,55 @@ $builder->joinedTableInheritance()->column('type');
 ```
 
```

Comment:
Perhaps good to update the index at the top of the page too. Not still refers to the callbacks only.